src/ui/mathliveview.js

```diff
@@ -0,0 +1,35 @@
+import InputView from '@ckeditor/ckeditor5-ui/src/view';
+import { Template } from '@ckeditor/ckeditor5-ui';
+
+export default class MathLiveView extends InputView {
+	constructor( locale, { options, attributes, style } ) {
+		super( locale );
+		this._options = options;
+		const bind = Template.bind( this, this );
+		this.set( 'value', '' );
+		this.setTemplate( {
+			tag: 'math-field',
+			attributes: {
+				...attributes,
+				style
```

Comment:
Why make an exception for styles instead of using just `attributes`?

Reply:
To space out styles. I believe this may improve readability.

Reply:
There are [a lot of possible arguments](https://cortexjs.io/docs/mathlive/#(MathfieldElement%3Aclass):~:text=its%20initial%20value.-,Attribute,Property,-disabled) we could expose here. I'm not sure why you made an exception for styles, especially that we use other arguments too.

I'm not going to push because it doesn't make much difference for us. But if I was a maintaner of ckeditor5-math and you opened a PR then I'd probably push to either enumerate all possible arguments or make no arbitrary exceptions.